pkgs/development/python-modules/ghp-import/default.nix

```
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, dateutil
+, fetchPypi
+, lib
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, dateutil
```
Conventionally, lib & buildPythonPackage & fetchpypi go first.

---

Done.